juriscraper/pacer/http.py

```diff
@@ -228,164 +230,76 @@ def _get_xml_view_state(r):
-                "loginForm:loginName": self.username,
-                "loginForm:password": self.password,
-                "loginForm:clientCode": "",
-                "loginForm": "loginForm",
-            },
-        )
-        if "Invalid username or password" in login_post_r.text:
-            raise PacerLoginException("Invalid username/password")
-        if "Username must be at least 6 characters" in login_post_r.text:
-            raise PacerLoginException(
-                "Username must be at least six " "characters"
-            )
-        if "Password must be at least 8 characters" in login_post_r.text:
-            raise PacerLoginException(
-                "Password must be at least eight " "characters"
-            )
-        if "timeout error" in login_post_r.text:
-            raise PacerLoginException("Timeout")
-
-        if not self.cookies.get("PacerSession"):
-            logger.info(
-                "Did not get cookies from first log in POSTs. "
-                "Assuming this is a filing user and doing two more "
-                "POSTs."
-            )
-            reg_msg_r = self.post(
-                url,
-                headers={"User-Agent": "Juriscraper"},
-                verify=False,
-                timeout=60,
-                auto_login=False,
-                data={
-                    "javax.faces.partial.ajax": "true",
-                    "javax.faces.source": "regmsg:chkRedact",
-                    "javax.faces.partial.execute": "regmsg:chkRedact",
-                    "javax.faces.partial.render": "regmsg:bpmConfirm",
-                    "javax.faces.partial.event": "change",
-                    "javax.faces.behavior.event": "valueChange",
-                    "javax.faces.ViewState": self._get_xml_view_state(
-                        login_post_r
-                    ),
-                    "regmsg": "regmsg",
-                    "regmsg:chkRedact_input": "on",
-                },
-            )
-            # The box is now checked. Submit the form to say so.
-            self.post(
-                url,
-                headers={"User-Agent": "Juriscraper"},
-                verify=False,
-                timeout=60,
-                auto_login=False,
-                data={
-                    "javax.faces.partial.ajax": "true",
-                    "javax.faces.source": "regmsg:bpmConfirm",
-                    "javax.faces.partial.execute": "@all",
-                    "javax.faces.ViewState": self._get_xml_view_state(
-                        reg_msg_r
-                    ),
-                    "regmsg": "regmsg",
-                    "regmsg:chkRedact_input": "on",
-                    "regmsg:bpmConfirm": "regmsg:bpmConfirm",
-                    "dialogName": "redactionDlg",
-                },
-            )
-
-        if not self.cookies.get("PacerSession") and not self.cookies.get(
-            "NextGenCSO"
+        response = login_post_r.json()
```

Comment:
Renaming the variable here and elsewhere would make the code a bit more clear:

```suggestion
        response_json = login_post_r.json()
```

It's very common to do something like this:

```
response = requests.get('https://some-url/')
```
So whenever I see a variable named `response`, I assume it's a response object, not a JSON object.